### PR TITLE
[DOCFIX] improve fsadmin doc

### PR DESCRIPTION
**Pull request by LuQQiu**
Update the fsadmin doc to include doctor command and make some modification.

### CODE REVIEW

**Comment by LuQQiu** [comment]
@aaudiber @apc999 Updated the fsadmin docs, PTAL, thanks!

File: docs/_data/table/fsadmin-command.csv [hunk 1]
```diff
@@ -1,3 +1,5 @@
 operation,syntax
-report,"report"
+backup,"backup [directory] [--local]"
+doctor,"doctor [category]""
```
Comment (aaudiber) [inline docs/_data/table/fsadmin-command.csv]: extra quote

File: docs/en/Admin-CLI.md [hunk 2]
```diff
@@ -91,3 +96,6 @@ write operations on the under storage.
 ```bash
 $ ./bin/alluxio fsadmin ufs --mode readOnly hdfs://ns
 ```
+
+The `fsadmin ufs` subcommand that takes the UFS URI as argument, the argument should be the root
```
Comment (aaudiber) [inline docs/en/Admin-CLI.md]: The `fsadmin ufs` subcommand takes a UFS URI as an argument. The argument should be a root UFS URI like ...

**Comment by LuQQiu** [comment]
@aaudiber Fixed! PTAL, thanks for finding out those issues.

**Comment by aaudiber** [comment]
@LuQQiu Please cherry-pick to branch-1.8

**Pull request merged**